libs/chain-signatures/contract/src/lib.rs

```diff
@@ -576,9 +577,7 @@ impl VersionedMpcContract {
     #[init(ignore_state)]
     #[handle_result]
     pub fn migrate() -> Result<Self, Error> {
-        if let Some(legacy_contract::VersionedMpcContract::V1(state)) =
-            env::state_read::<legacy_contract::VersionedMpcContract>()
-        {
+        if let Some(state) = env::state_read::<legacy_contract_state::MpcContractV1>() {
```

Comment:
I think we need to deserialize the previous [`VersionedMpcContract`](https://github.com/Near-One/mpc/blob/53f549cf5a58b9072b2f5a13d2e96c78e70ebc7d/libs/chain-signatures/contract/src/lib.rs#L58).